pkg/domain/msgResp.go

```diff
@@ -0,0 +1,7 @@
+package domain
+
+type MsgResp struct {
```

Comment:
what is the purpsoe of this struct? It looks like the only field that is used is the value. Is this struct needed or are there plans for using the other values in later changes?